pkgs/applications/networking/cluster/k3s/docs/USAGE.md

```diff
@@ -0,0 +1,58 @@
+# K3s Usage
+
+## Single Node
+
+```
+{
+  networking.firewall.allowedTCPPorts = [
+    6443 # k3s: required so that pods can reach the API server (running on port 6443 by default)
+    # 2379 # k3s, etcd clients: required if using a "High Availability Embedded etcd" configuration
+    # 2380 # k3s, etcd peers: required if using a "High Availability Embedded etcd" configuration
+  ];
+  networking.firewall.allowedUDPPorts = [
+    # 8472 # k3s, flannel: required if using multi-node for inter-node networking
+  ];
+  services.k3s.enable = true;
+  services.k3s.role = "server";
+  services.k3s.extraFlags = toString [
+    # "--kubelet-arg=v=4" # Optionally add additional args to k3s
+  ];
+  environment.systemPackages = [ pkgs.k3s ];
```

Comment:
This is done by the nixos module already: https://github.com/NixOS/nixpkgs/blob/22b57baf0987b044750398babf0230a0736c5eec/nixos/modules/services/cluster/k3s/default.nix#L155

```suggestion
```